web/usuaris/src/components/organisms/SideMenu.tsx

```diff
@@ -1,39 +1,133 @@
 import { FC } from 'react'
-import styled from 'styled-components'
 import { FlexBox, colors, dimensions, font } from '@itacademy/ui'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
 import { icons } from '../../assets/icons'
 
+const LinkCategory = styled(Link)`
+  color: ${colors.gray.gray3};
+  font-size: ${font.xs};
+  font-weight: 600;
+  font-family: ${font.fontFamily};
+  cursor: pointer;
+  text-decoration: none;
+
+  &:hover {
+    color: ${colors.primary};
+  }
+`
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: auto 1fr;
+  gap: ${dimensions.spacing.xxxs};
+  align-items: center;
+`
+
 const SideMenuStyled = styled(FlexBox)`
   height: 100%;
+  min-width: 12rem;
   margin-top: ${dimensions.spacing.lg};
-  color: ${colors.black.black3};
 
   ul {
     list-style: none;
-    padding: ${dimensions.spacing.sm};
+    padding: ${dimensions.spacing.xxxs};
   }
 
   li {
     font-size: ${font.xs};
     font-weight: ${font.medium};
     margin-bottom: ${dimensions.spacing.xxl};
+
+    &::before {
+      font-size: larger;
+      color: ${colors.primary};
+      margin-right: 0.3rem;
+    }
+  }
+`
+
+const CategoryStyled = styled.span`
+  color: ${colors.gray.gray3};
+  font-size: ${font.xs};
+  font-weight: 600;
+  font-family: ${font.fontFamily};
+  cursor: pointer;
+  transition: transform 0.3s ease;
+  outline: none;
+  display: inline-block;
+  border: 0px solid black;
+  tabindex: 0;
+
+  &:hover {
+    color: ${colors.primary};
+    transform: scale(1.05);
+  }
+
+  &:focus {
+    color: ${colors.black.black1};
+  }
+
+  &:focus::before {
+    content: '●';
+    font-size: larger;
+    color: ${colors.primary};
+    margin-right: 0.3rem;
   }
 `
 
+const ImgStyled = styled.img`
+  height: 30px;
+  margin-right: ${dimensions.spacing.xxxs};
+`
+
 const LogoImage = styled.img`
   max-width: 7.25rem;
   height: auto;
 `
 
+const menu = [
+  {
+    id: 1,
+    title: 'Usuari@s',
```

Comment:
Yes, that is indeed the idea. However, this is a temporary proposal since the previous code was hard-coded, and I want to ask daily if we will proceed with an API or something like that.

Thanks for the feedback!

Reply:
https://dev.sso.itawiki.eurecatacademy.org/api/v1/api-docs

Reply:
Hecho